### PR TITLE
[12.0][IMP][account_invoice_supplier_duplicate_ref] module creation

### PR DESCRIPTION
**Pull request by Tonow-c2c**
This module deactivate the odoo check for unique reference on supplier invoice reference.

Normal behaviour : 
BSTRS-79 2019-06-05 17-16

With the module installed : 
BSTRS-79 done 2019-06-05 17-37

### CODE REVIEW

**Review by vrenaville — APPROVED** [review]
Thanks for the work

File: account_invoice_supplier_duplicate_ref/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (asaunier) [inline account_invoice_supplier_duplicate_ref/__init__.py]:
```suggestion
```

File: account_invoice_supplier_duplicate_ref/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+This module allows to validate supplier invoices that have the same reference number.
+Leasing invoices, for example, which are the same each month and whose BVR number does not change.
```
Comment (asaunier) [inline account_invoice_supplier_duplicate_ref/readme/DESCRIPTION.rst]:
```suggestion
Leasing invoices, for example, which are the same each month and whose ISR number does not change.
```

File: account_invoice_supplier_duplicate_ref/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+
+This module deactivate the odoo check for unique reference on supplier invoice reference.
```
Comment (asaunier) [inline account_invoice_supplier_duplicate_ref/readme/DESCRIPTION.rst]:
```suggestion
This module deactivates the odoo check for unique reference on supplier invoice reference.
```

File: account_invoice_supplier_duplicate_ref/readme/ROADMAP.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (asaunier) [inline account_invoice_supplier_duplicate_ref/readme/ROADMAP.rst]:
```suggestion
```

File: account_invoice_supplier_duplicate_ref/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+* Install the module
+* Go one on Accounting/Vendor/Bill
```
Comment (asaunier) [inline account_invoice_supplier_duplicate_ref/readme/USAGE.rst]:
```suggestion
* Go on Accounting/Vendor/Bill
```

File: account_invoice_supplier_duplicate_ref/tests/test_account_invoice_supplier_duplicate_ref.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+# TODO:
```
Comment (asaunier) [inline account_invoice_supplier_duplicate_ref/tests/test_account_invoice_supplier_duplicate_ref.py:4]: ?

Comment (Tonow-c2c) [reply account_invoice_supplier_duplicate_ref/tests/test_account_invoice_supplier_duplicate_ref.py:4]: I add this section [https://github.com/OCA/account-invoicing/pull/535/files#diff-a2bb309a44ef9db6153ec476db4d06cfR2](https://github.com/OCA/account-invoicing/pull/535/files#diff-a2bb309a44ef9db6153ec476db4d06cfR2)

**Review by i-vyshnevska — CHANGES_REQUESTED** [review]
this addon totally duplicates functional from https://github.com/OCA/account-invoicing/tree/12.0/account_invoice_supplier_ref_reuse

**Comment by Tonow-c2c** [comment]
Thanks @i-vyshnevska you totally right ! 
I pushed before but not the other PR is already merged  \o/

**Pull request closed without merging**